packages/perseus-core/src/types.ts

```diff
@@ -1,6 +1,7 @@
 // Types that can be shared between Perseus packages
 // ideally without causing circular dependencies
 
+// TODO: this should be typed
 type State = any;
```

Comment:
Originally I changed this to `type State = unknown;` to trigger Changeset, but then I worried that that would break other things so I just added a comment. Will that still let me cut a release?

Reply:
I have no idea. We'll have to try and see.